src/System.Net.Http/src/System.Net.Http.builds

```diff
@@ -12,6 +12,10 @@
       <OSGroup>Windows_NT</OSGroup>
       <TargetGroup>netcore50</TargetGroup>
     </Project>
+    <Project Include="System.Net.Http.csproj">
+      <OSGroup>Windows_NT</OSGroup>
```

Comment:
We likely aren't consistent but either works fine for net4x targets. If OSGroup is excluded this will be build as AnyOS, if this depends on Windows functionality, which I suspect it does then marking it as Windows_NT is more correct.